src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,14 @@
+.catalog {
+  max-width: 1200px;
+  min-width: 1024px;
+  margin: 0 auto;
```

Comment:
I'm not sure whether you need the styles, it'll look the same

Reply:
got you, done